images/nginx-1.25/rootfs/build.sh

```diff
@@ -41,8 +41,8 @@ export MODSECURITY_VERSION=v1.0.3
 # Check for recent changes: https://github.com/SpiderLabs/ModSecurity/compare/v3.0.8...v3/master
 export MODSECURITY_LIB_VERSION=v3.0.12
 
-# Check for recent changes: https://github.com/coreruleset/coreruleset/compare/v3.3.5...main
-export OWASP_MODSECURITY_CRS_VERSION=v3.3.5
+# Check for recent changes: https://github.com/coreruleset/coreruleset/compare/v3.3.5...v4.0/main
```

Comment:
Is this supposed to be 
```
# Check for recent changes: https://github.com/coreruleset/coreruleset/compare/...v4.0/main
```

Reply:
Tao or others would have commented or may comment if it has any impact but its obviously just a nit so nto a big deal at all.

The current line does not highlight it the way you are suggesting. But I don't know better as I was only looking at the current line and not the  content of that url, if I would do a real comparison on github web ui. I will check now if that a real practical github webui comparison tool url is formatted like that (with the old + the new version). If you already know that it really is how that github compare tool is in real life, then thanks for fixing this nit/comment.

Reply:
After checking on github gui, I see the URL is  https://github.com/coreruleset/coreruleset/compare/v3.3/master...v4.0/main  , it looks like this 
![image](https://github.com/kubernetes/ingress-nginx/assets/5085914/cd9db72b-d190-440c-af15-ef6044896be6)

Reply:
ok. thank you. settles it. no more changes needed. and once again, thank you very much for this contribution. cheers